src/cmd/main.cpp

```diff
@@ -166,9 +167,13 @@ int main(int argc, const char* argv[]) {
       return ERROR_IN_COMMAND_LINE;
     }
 
-    INIT_FILE_LOGGER("zilliqa", logpath.c_str());
-    INIT_STATE_LOGGER("state", logpath.c_str());
-    INIT_EPOCHINFO_LOGGER("epochinfo", logpath.c_str());
+    if (vm.count("stdoutlog")) {
+      INIT_STDOUT_LOGGER();
```

Comment:
Can you double-check what happens to calls to `LOG_STATE` and `LOG_EPOCH`?

Reply:
You are correct. Thanks for the observation.The state logs and epochinfo logs were going into common log file instead of `stdout`.
I changed code and initialize all three types of logs in `INIT_STDOUT_LOGGER` macro.Below are the stdout captured logs from zilliqa process.
`StateLog:
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:664][ProcessVCDSBlocksMes] New DS leader is at 1
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:678][ProcessVCDSBlocksMes] [Epoch 140] DS BACKUP NOW
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:680][ProcessVCDSBlocksMes] [Epoch I am now DS backup for the next round] 140
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:176][UpdateGovProposalRem] BEG
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:176][UpdateGovProposalRem] END
[ 2078][20-09-15T11:08:34.238][stProcessing.cpp:493][StartFirstTxEpoch   ] BEG

EpochInfo Log:
[ 2071][20-09-15T10:49:26.333][work/P2PComm.cpp:413][ClearBroadcastHashAs] END
[ 20-09-15T10:49:26.333 ][BROAD][<0000000000000018.236.149.122:33133>][F78462] RECV
[ 2100][20-09-15T10:49:26.333][ckProcessing.cpp:469][ProcessMicroblockSub] BEG

Epoch Log:
[ 2078][20-09-15T11:08:34.238][ckProcessing.cpp:680][ProcessVCDSBlocksMes] [Epoch I am now DS backup for the next round] 140`